src/parser/metrics/RealLinesOfCode.ts

```diff
@@ -81,6 +96,24 @@ export class RealLinesOfCode implements Metric {
         };
     }
 
+    isComment(node: Parser.SyntaxNode) {
+        return this.commentStatementsSet.has(node.type);
+    }
+
+    isPythonComment(node: Parser.SyntaxNode) {
+        return this.isComment(node) || this.isPythonMultilineComment(node);
+    }
+
+    isPythonMultilineComment(node: Parser.SyntaxNode) {
+        // Multiline comments in python are (multiline) strings that are
+        // neither assigned to a variable nor used as call parameter.
+        return (
+            node.type === "expression_statement" &&
+            node.childCount === 1 &&
+            node.child(0)?.type === "string"
+        );
```

Comment:
parenthesis are not needed :-)

Reply:
Oh, I was sure I removed them. How embarrassing. 
As My Linh already merged this, I have to open a new PR for this.